CODE-OF-CONDUCT.md

```diff
@@ -55,7 +55,7 @@ further defined and clarified by project maintainers.
 ## Enforcement
 
 Instances of abusive, harassing, or otherwise unacceptable behaviour may be
-reported by contacting [a member of the project team](PUT_YOUR_TEAM_CONTACT_DETAILS_HERE). All
+reported by contacting [the project team](oss@expediagroup.com). All
```

Comment:
Currently we ask teams to replace the placeholder with their GitHub Team.
I think adding this email address makes more sense. But we might be spammed even more from bots :(

Reply:
That was the direction I got when releasing my project thus it ended up in here.